From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 19 Feb 2023 23:37:22 +0100
Subject: [PATCH] Change includes <sys/poll.h> to <poll.h>

Fix the following warning:

    #warning redirecting incorrect #include <sys/poll.h> to <poll.h>

--- a/mavis/mavis.h
+++ b/mavis/mavis.h
@@ -16,3 +16,3 @@
 #include <sys/time.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/types.h>
--- a/mavis/spawnd_headers.h
+++ b/mavis/spawnd_headers.h
@@ -35,3 +35,3 @@
 #include <sys/wait.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <setjmp.h>
--- a/mavisd/headers.h
+++ b/mavisd/headers.h
@@ -38,3 +38,3 @@
 
-#include <sys/poll.h>
+#include <poll.h>
 
--- a/misc/io_sched.c
+++ b/misc/io_sched.c
@@ -14,3 +14,3 @@
 #include <sys/types.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/time.h>
@@ -42,3 +42,3 @@
 #ifdef WITH_POLL
-#include <sys/poll.h>
+#include <poll.h>
 #endif
--- a/tcprelay/headers.h
+++ b/tcprelay/headers.h
@@ -35,3 +35,3 @@
 #include <sys/wait.h>
-#include <sys/poll.h>
+#include <poll.h>
 
